test/cppsuite/configs/schema_sweep_default.txt

```diff
@@ -0,0 +1,34 @@
+# Configuration for schema_sweep.
+duration_seconds=5,
+cache_size_mb=250,
+workload_manager=
+(
+    checkpoint_config=
+    (
+        op_rate=5s,
```

Comment:
You should make this <5s so it is guaranteed to fire at least once in the default config.

Reply:
The default config is almost irrelevant for this test and I don't want to have a too high rate for the checkpoint. Not sure what I should do? Happy to disable it for the default configuration.

Reply:
The intent behind default configs was to make a condensed version of the test that tests all relevant code paths, if you think checkpoint isn't relevant i'm happy for you to disable it but otherwise having it be 1s shouldn't impact your test should it?

Reply:
I have updated the default cfg here 4708d64f5154835eed8583a6a707ac09207076fe. Let me know if this works for you.